FileShareClient/Models/SearchBatchAttribute.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+
+namespace UKHO.FileShareClient.Models
+{
+    public class SearchBatchAttribute 
+    {
+        public SearchBatchAttribute(string key = default, List<string> values = default)
+        {
+            Key = key;
+            Values = values;
+        }
+
+        [DataMember(Name = "key", EmitDefaultValue = false)]
+        public string Key { get; set; }
+
+        [DataMember(Name = "values", EmitDefaultValue = false)]
+        public List<string> Values { get; set; }
+
+        public override string ToString()
+        {
+            var sb = new StringBuilder();
+            sb.Append($"class {nameof(SearchBatchAttribute)} {"{"}\n");
+            sb.Append($"{nameof(Key)}: {(Key)}\n");
```

Comment:
Please reinstate the indentation before the field names
```suggestion
            sb.Append($"  {nameof(Key)}: {(Key)}\n");
```
Please apply to all fields

Reply:
Updated as suggested. Thanks